plugin/src/main/java/org/opensearch/ml/utils/MLExceptionUtils.java

```diff
@@ -59,4 +59,8 @@ public static void logException(String errorMessage, Exception e, Logger log) {
             log.error(errorMessage, e);
         }
     }
+
+    public static Throwable getRootCause(Throwable t) {
```

Comment:
seems not necessary to create a new method for one line of code

Reply:
See my below comment please.